src/components/non-fixed/ChatBody/OneDateContainer/OneDateContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import { useRecoilState } from 'recoil';
+import DateDivider from '@components/non-fixed/ChatBody/DateDivider/DateDivider';
+import { userNumberState, messageDataState } from '@context/state/atom';
+import ChatLog from '@components/non-fixed/ChatBody/ChatLog/ChatLog';
+
+const StyledOneDateContainer = styled.div`
+  width: 100%;
+  height: auto;
+`;
+
+export default function OneDateContainer({
+  messageDate,
+}: {
+  messageDate: string;
+}) {
+  const [userNumber, setUserNumber] = useRecoilState(userNumberState);
+  const [messageData, setMessageData] = useRecoilState(messageDataState);
+
+  const oneDateMessagData = messageData[messageDate];
+
+  return (
+    <StyledOneDateContainer>
+      <DateDivider dateString={messageDate} />
+      {oneDateMessagData.map((data) => {
+        return (
+          <ChatLog
+            isEqual={userNumber === data.from}
+            from={data.from}
+            createdAt={data.createdAt}
+            content={data.content}
+            key={data.createdAt}
+            like={data.like}
+          />
+        );
+      })}
```

Comment:
이렇게 배열 데이터를 순회하면서 컴포넌트에 mapping해주는 일을 앞으로 정말 많이 하게 될 텐데, 대부분의 경우 이번 처럼 다른 로직을 필요로 하지 않고 매핑할 컴포넌트만 반환해준답니당

이럴 땐 중괄호와 return문을 생략하고 바로 data.map((item,idx)=> <ChatLog>...</ChatLog>) 이렇게 표현해주면 간결해지는데요, return 값만 있을 경우 return 문을 생략할 수 있는건 ES6의 화살표 함수 개념이니까 참고하세요 > <

📚 [MDN 문서](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

```suggestion
      {oneDateMessagData.map((data) => {
          <ChatLog
            isEqual={userNumber === data.from}
            from={data.from}
            createdAt={data.createdAt}
            content={data.content}
            key={data.createdAt}
            like={data.like}
          />
      })}
```

Reply:
역시나 바닐라JS가 항상 중요하단 것을 느끼고 갑니다!